docs/poly-apis.md

```diff
@@ -0,0 +1,636 @@
+
+The  32-bit unsigned integer type encoded as the 0x-prefixed hex string in JSON.
+
+#### Examples
+
+|  JSON | Decimal Value |
+| --- |--- |
+|  “0x0” | 0 |
+|  “0x10” | 16 |
+|  “10” | Invalid, 0x is required |
+
+### Type `Uint64`
+
+The  64-bit unsigned integer type encoded as the 0x-prefixed hex string in JSON.
+
+#### Examples
+
+|  JSON | Decimal Value |
+| --- |--- |
+|  “0x0” | 0 |
+|  “0x10” | 16 |
+|  “10” | Invalid, 0x is required |
+
+### Type `H256`
+
+The 32-byte fixed-length binary data.
+
+The name comes from the number of bits in the data.
+
+In JSONRPC, it is encoded as a 0x-prefixed hex string.
+
+#### Examples
+
+```
+0x696447c51fdb84d0e59850b26bc431425a74daaac070f2b14f5602fbb469912a
+```
+
+
+### Type `PolyVersionInfo`
+
+
+#### Fields
+
+`PolyVersionInfo` is a JSON object with the following fields.
+
+*   `versions`: [`Versions`](#type-versions)
+
+*   `nodeInfo`: [`NodeInfo`](#type-nodeinfo)
+
+
+### Type `BackendInfo`
+
+
+#### Fields
+
+`BackendInfo` is a JSON object with the following fields.
+
+*   `validatorCodeHash`: [`H256`](#type-h256) - Validator script's code hash
+
+*   `generatorCodeHash`: [`H256`](#type-h256) - Generator script's code hash
+
+*   `validatorScriptTypeHash`: [`H256`](#type-h256) - Validate script's hash (script.hash_type = type)
+
+
+### Type `ScriptInfo`
+
+
+#### Fields
+
+`ScriptInfo` is a JSON object with the following fields.
+
+*   `typeHash`: [`H256`](#type-h256) - Hash of script
+
+*   `script`: [`Script`](#type-script) - Script's deploy info
+
+
+### Type `Script`
+
+Describes the lock script and type script for a cell.
+
+#### Examples
+
+
+```
+{
+  "code_hash": "0x28e83a1277d48add8e72fadaa9248559e1b632bab2bd60b27955ebc4c03800a5",
+  "hash_type": "data",
+  "args": "0x"
+}
+```
+
+
+#### Fields
+
+`Script` is a JSON object with the following fields.
+
+*   `code_hash`: [`H256`](#type-h256) - The hash used to match the script code.
+
+*   `hash_type`: [`ScriptHashType`](#type-scripthashtype) - Specifies how to use the `code_hash` to match the script code.
+
+*   `args`: [`JsonBytes`](#type-jsonbytes) - Arguments for script.
+
+
+### Type `ScriptHashType`
+
+Specifies how the script `code_hash` is used to match the script code and how to run the code.
+
+Allowed kinds: "data", "type" and "data1".
+
+Refer to the section [Code Locating](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#code-locating) and [Upgradable Script](https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0022-transaction-structure/0022-transaction-structure.md#upgradable-script) in the RFC *CKB Transaction Structure*.
+
+`ScriptHashType` is equivalent to `"data" | "type" | "data1"`.
+
+*   Type "data" matches script code via cell data hash, and run the script code in v0 CKB VM.
+*   Type "type" matches script code via cell type script hash.
+*   Type "data" matches script code via cell data hash, and run the script code in v1 CKB VM.
```

Comment:
The reference links are enough, in my opinion. We don't need to re-explain the content in the referenced documents.

Another useful RFC is https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0032-ckb-vm-version-selection/0032-ckb-vm-version-selection.md

Reply:
Updated